lib/config/hadeployconfigverify.go

```diff
@@ -211,7 +211,7 @@ func (c *HaDeployConfig) verifyAutomateSettings() error {
 	if err := validateNumberField(automateSettings.InstanceCount, "automate instance_count", true); err != nil {
 		errorList.PushBack(err)
 	}
-	if err := validateRequiredString(automateSettings.ConfigFile, "config_file", "configs/automate.toml"); err != nil {
+	if err := validateRequiredString(automateSettings.ConfigFile, "config_file", "configs/automate.toml", "/hab/a2_deploy_workspace/configs/automate.toml"); err != nil {
 		errorList.PushBack(err)
```

Comment:
Don't check for any particular value, just add the check for empty fields.

Reply:
Done